core/src/language/python.rs

```diff
@@ -621,6 +458,318 @@ impl Python {
         };
         Ok(())
     }
+
+    fn gen_unit_variant_ctor(
```

Comment:
instead of ctor can we put constructor just so we dont get confused and that goes for all functions.